components/script/script_thread.rs

```diff
@@ -146,6 +146,8 @@ struct InProgressLoad {
     clip_rect: Option<Rect<f32>>,
     /// Window is frozen (navigated away while loading for example).
     is_frozen: bool,
+    /// Window is not visible.
+    is_hidden: bool,
```

Comment:
Nit. Maybe you want to name that `is_visible`, like the other properties added to pipeline, pipeline_details and iframe .